cli/src/main/kotlin/com/malinskiy/marathon/cli/config/deserialize/StrictModeDeserializer.kt

```diff
@@ -0,0 +1,16 @@
+package com.malinskiy.marathon.cli.config.deserialize
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.malinskiy.marathon.execution.strategy.StrictMode
+
+class StrictModeDeserializer : StdDeserializer<StrictMode>(StrictMode::class.java) {
+    override fun deserialize(
+        p: JsonParser,
+        ctxt: DeserializationContext
+    ): StrictMode {
+        val value = p.valueAsString.toUpperCase()
```

Comment:
Need to check what user will get in console

Reply:
This deserializer has be compatible with the old version: false + true and print a deprecation message in such case